Makefile

```diff
@@ -106,7 +106,7 @@ create-db-snapshot-service: ## Create a db service from the latest db snapshot
 
 .PHONY: check-db-snapshot-service
 check-db-snapshot-service: ## Get the status for the db snapshot service
-	@cf service digitalmarketplace_api_db_snapshot | grep -i 'status: ' | sed 's/^.*: //' | awk '{print toupper($0)}'
+	@cf service digitalmarketplace_api_db_snapshot | grep -i 'status: ' | sed 's/^.*: //' | tr '[:lower:]' '[:upper:]'
```

Comment:
This was the first time I heard of/used it. Stack overflow is a wonderful thing.